scripts/2017/interactive-bots.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+from upload import upload_task
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--force', dest='force', action="store_true", default=False)
+args = parser.parse_args()
+
+bots_to_create = [
+    ('Chess', ('lets you play chess against the computer or other people on Zulip')),
+    ('Merels', ('lets you play merels against the computer or other people on Zulip')),
+    ('Four in a row', ('lets you play four in a row against the computer or other people on Zulip')),
+    ('Google Hangouts', ('posts a link to a Hangouts call.')),
+    ('lunch', ('lets users create and join lunch events')),
+    ('URL shortener', ('takes a link and sends back a shortened version using a shortener service')),
```

Comment:
In terms of project priorities, I think we care a lot more about implementing Zulip bot versions of popular hubot/slack/errbot integrations (especially hubot, since it's kinda a pain), than writing new games.  There's 2 reasons for this:
* We need to make fewer decisions when porting an existing bot implementation, so less mentor coordination is required.
* The results will be bots we're confident our users will find useful, since they replicate functionality available with other chat apps.

So to make the best use of everyone's time, I think we should probably make this list be 80% "integrations we expect people to use" and 20% "write a game", rather than the other way around.  

Can you browser around and try to make this list have another 15 things chosen because they're popular integrations elsewhere?

Reply:
Yeah, will add some more. I also have a bunch of integrations on my list left, but these are more fitting for the "webhook integrations" task.

Reply:
Cool.  It'd be reasonable for you to do a pass on the "webhook integrations" script to fill those in now, even though we haven't finished updating the task yet.